### PR TITLE
change mkdir --parent to mkdir -p

### PR DESCRIPTION
**Pull request by u--**
###### Reviewer checklist

- [ ] Test coverage: `stack test --coverage`
- [ ] Public API documentation: `stack haddock`
- [ ] Style conformance: `stylish-haskell`

---

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ k-frontend:
 	rm -rf $(K_DIR) $(K_NIGHTLY)
 	curl --location --output $(K_NIGHTLY) \
 	    $$(curl 'https://api.github.com/repos/kframework/k/releases' | jq --raw-output '.[0].assets[0].browser_download_url')
-	mkdir --parents $(K_DIR)
```
[the remark this answers is not in the document]
Comment (dwightguth) [reply Makefile]: Judging from the branch name on this pull request I would guess that mkdir --parent isn't POSIX-compliant and therefore doesn't work on OSX.

[the remark this answers is not in the document]
Comment (u--) [reply Makefile]: Yeah, it's stupid but why not fix it.

[the remark this answers is not in the document]
Comment (virgil-serbanuta) [reply Makefile]: Thank you!

**Comment by ttuegel** [comment]
Jenkins: test this please

**Pull request merged**